components/tidb_query/src/codec/mysql/duration.rs

```diff
@@ -646,25 +647,7 @@ impl Duration {
     /// Converts a `Duration` to printable numeric string representation
     #[inline]
     pub fn to_numeric_string(self) -> String {
```

Comment:
I think we could remove this method.

Reply:
No, `to_numeric_string` is different from `to_string`, `to_string` has sep `:`, to_numeric_string` has no sep.

Reply:
I mean use `self.format("")` to replace `to_numeric_string`, not the `to_string`.

Reply:
I think keep this function will make the semantics clear (in where caller call this function )